.github/workflows/beam_Metrics_Report.yml

```diff
@@ -18,10 +18,10 @@
 name: Beam Metrics Report
 
 on:
-  # issue_comment:
-  #   types: [created]
-  # schedule:
-  #   - cron: '0 */23 * * *'
+  issue_comment:
```

Comment:
probably do not need issue comment trigger? (these result in an email to be sent and should be triggered by committer only and can be done with workflow_dispatch

Reply:
You're right - thanks, fixed